src/modules/files.nix

```diff
@@ -0,0 +1,58 @@
+{ pkgs, lib, ... }:
+
+let
+  types = lib.types;
+  ini = pkgs.formats.ini { };
```

Comment:
For editorconfig, you'll need https://github.com/nix-community/nixpkgs.lib/blob/master/lib/generators.nix#L164